scripts/karma.coffee

```diff
@@ -138,6 +138,19 @@ class Karma
     sorted = @sort()
     sorted.slice(-n).reverse()
 
+  notify: (msg, subject, user, verb) ->
+    if msg.message.rawMessage.channel._modelName == 'DM'
```

Comment:
There's got to be a more elegant way to do this right? switch/case? Even better to have a dictionary type object that looks up the response based on the _modelName.